tests/ui/sync/mutexguard-sync.stderr

```diff
@@ -7,6 +7,7 @@ LL |     test_sync(guard);
    |     required by a bound introduced by this call
    |
    = help: the trait `Sync` is not implemented for `Cell<i32>`
+   = note: if you want to do aliasing and mutation between multiple threads, use `std::sync::RwLock`
    = note: if you want to do aliasing and mutation between multiple threads, use `std::sync::RwLock` or `std::sync::atomic::AtomicI32` instead
```

Comment:
Huh, can you adjust the standard library impls to deduplicate these?

Reply:
I've pushed https://github.com/rust-lang/rust/pull/117205/commits/3c7daf9d945ba81446a4aac4ab79a634d2ef1f50 to address that.